src/types.ts

```diff
@@ -19,41 +19,46 @@ export declare type TwentyToFortyEight = NumericRange<
 >;
 
 export declare type MaterialSymbolsOptions = {
-  /**
-   * Variant of the symbol icons.
-   */
-  variant?: 'outlined' | 'rounded' | 'sharp';
   /**
    * Styles of the symbol icons.
    */
-  styling?: Partial<StylingOptions>;
+  styling: Partial<StylingOptions>;
   /**
-   * Maybe we could remove that? and force a name instead.
+   * Placeholders let you inline symbols when the prefix
+   * is encounter.
+   * For instance `@md_symbol_info` in your html will be replaced
```

Comment:
in your html and or js to be complete

Reply:
no by html I mean template, not file extensions.

Reply:
but now I think of it, shouldn't it be `@md-symbol-info` ?

Reply:
> but now I think of it, shouldn't it be `@md-symbol-info` ?

yes prob better less confusing

Reply:
I think hyphens are actually confusing, I wrote underscore naturally because it's less common to see underscores in html, html tags use hyphens already and `md-icon` elements uses underscore e.g. `<md-icon>remove_red_eyes</md-icon>`. But that means the code needs to be updated.